src/main/views/common/correspondence-address-confirm/correspondence-address-confirm.njk

```diff
@@ -3,7 +3,12 @@
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
       <form class="form" action="" method="post">
-        {% include "partials/user_name.njk" %}
+         {% if businessName %}
```

Comment:
you also need to add acspType in the if condition for both businessName and firstName
acspType: acspData?.typeofBusiness